R/satterthwaite.R

```diff
@@ -173,29 +173,14 @@ h_df_1d_list <- function(est,
 
 #' Calculation of Satterthwaite Degrees of Freedom for One-Dimensional Contrast
 #'
-#' @description `r lifecycle::badge("experimental")`
-#'
 #' @param object (`mmrm`)\cr the MMRM fit.
 #' @param contrast (`numeric`)\cr contrast vector. Note that this should not include
 #'   elements for singular coefficient estimates, i.e. only refer to the
 #'   actually estimated coefficients.
 #'
 #' @return List with `est`, `se`, `df`, `t_stat` and `p_val`.
-#' @export
-#'
-#' @examples
-#' object <- mmrm(
-#'   formula = FEV1 ~ RACE + SEX + ARMCD * AVISIT + us(AVISIT | USUBJID),
-#'   data = fev_data
-#' )
-#' contrast <- numeric(length(object$beta_est))
-#' contrast[3] <- 1
-#' df_1d(object, contrast)
-df_1d <- function(object, contrast) {
-  assert_class(object, "mmrm")
-  assert_numeric(contrast, any.missing = FALSE)
-
-  contrast <- as.vector(contrast)
+#' @keywords internal
+df_1d_sat <- function(object, contrast) {
```

Comment:
ok if this is internal then no lifecycle badge is good, but then prefix with `h_`